web/src/user_groups.ts

```diff
@@ -325,7 +333,7 @@ function get_display_name_for_system_group_option(setting_name: string, name: st
 export function check_system_user_group_allowed_for_setting(
     group_name: string,
     group_setting_config: GroupPermissionSetting,
-    for_new_settings_ui = false,
+    for_new_settings_ui: boolean,
```

Comment:
`for_new_settings_ui` is probably a name that we should plan to be temporary; "new" loses its meaning over time. But I'm not sure what to replace it with, so it can stay for now..